Makefile

```diff
@@ -24,25 +24,27 @@ docker-run-currentweather: ## Running your custom-built docker image locally
 docker-push: ## Pushing the freshly built image to the registry
 	docker push $(DOCKER_USERNAME)/currentweather-nodejs
 
-docker-clean: ## Remove the stuff we built locally afterwards
+docker-stop: ## Remove the stuff we built locally afterwards
 	docker kill redis-container
 	docker rmi -f $(DOCKER_USERNAME)/currentweather-nodejs || true
 	docker network rm currentweather_nw || true
 
-kube-create: ## Create kubernetes rc and svc 
+kube-run: ## Create kubernetes rc and svc
 	kubectl create -f redis-rc.yml
 	kubectl create -f redis-svc.yml
 	kubectl create -f currentweather-cm.yml
 	kubectl create -f currentweather-rc.yml
 	kubectl create -f currentweather-svc.yml
 
-kube-delete: ## Delete rc, cm and svc
+kube-stop: ## Delete rc, cm and svc
+	kubectl scale --replicas=0 rc redis
 	kubectl delete -f redis-rc.yml
```

Comment:
It seems to be good kubernetes practice, source unavailable: first scale
down to 0 than delete

Am Samstag, 16. April 2016 schrieb Matthias Lübken :

> In Makefile
> <https://github.com/luebken/currentweather/pull/5#discussion_r59964604>:
>
> >  	kubectl create -f redis-rc.yml
> >  	kubectl create -f redis-svc.yml
> >  	kubectl create -f currentweather-cm.yml
> >  	kubectl create -f currentweather-rc.yml
> >  	kubectl create -f currentweather-svc.yml
> >
> > -kube-delete: ## Delete rc, cm and svc
> > +kube-stop: ## Delete rc, cm and svc
> > +	kubectl scale --replicas=0 rc redis
> >  	kubectl delete -f redis-rc.yml
>
> Why is the scale down needed?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/luebken/currentweather/pull/5/files/0207e4020887de310e821bb406ee9db7443ea360#r59964604>
>


-- 
Principal Software Engineer - Systems Design & Engineering
Mobile: +49 171 2801345

Red Hat GmbH, http://www.de.redhat.com/, Sitz: Grasbrunn,
Handelsregister: Amtsgericht München, HRB 153243,
Geschäftsführer: Paul Argiry, Charles Cachera, Michael Cunningham, Michael
O'Neill